src/Compilers/VisualBasic/Test/Symbol/SymbolDisplay/SymbolDisplayTests.vb

```diff
@@ -5318,6 +5318,113 @@ ref struct S<T>
                 SymbolDisplayPartKind.TypeParameterName)
         End Sub
 
+        ''' <summary>
+        ''' IParameterSymbol.IsRefScoped and IsValueScoped are ignored in VisualBasic.SymbolDisplayVisitor.
+        ''' </summary>
+        <Theory>
+        <InlineData(False)>
+        <InlineData(True)>
+        Public Sub ScopedParameter(includeScoped As Boolean)
+            Dim source =
+"ref struct R { }
+class Program
+{
+    static void F(scoped R r1, in scoped R r2, scoped ref R r3) { }
+}"
+            Dim comp = CreateCSharpCompilation(GetUniqueName(), source, parseOptions:=New CSharp.CSharpParseOptions(CSharp.LanguageVersion.Preview))
+            comp.VerifyDiagnostics()
+            Dim method = comp.GlobalNamespace.GetTypeMembers("Program").Single().GetMembers("F").Single()
+
+            Dim format = SymbolDisplayFormat.TestFormat
+            If includeScoped Then
+                format = format.AddParameterOptions(SymbolDisplayParameterOptions.IncludeScoped)
+            End If
+
+            Verify(SymbolDisplay.ToDisplayParts(method, format),
+                "Sub Program.F(r1 As R, r2 As R, r3 As R)",
+                SymbolDisplayPartKind.Keyword,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.ClassName,
+                SymbolDisplayPartKind.Operator,
+                SymbolDisplayPartKind.MethodName,
+                SymbolDisplayPartKind.Punctuation,
+                SymbolDisplayPartKind.ParameterName,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.Keyword,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.StructName,
+                SymbolDisplayPartKind.Punctuation,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.ParameterName,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.Keyword,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.StructName,
+                SymbolDisplayPartKind.Punctuation,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.ParameterName,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.Keyword,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.StructName,
+                SymbolDisplayPartKind.Punctuation)
+        End Sub
+
+        ''' <summary>
+        ''' ILocalSymbol.IsRefScoped and IsValueScoped are ignored in VisualBasic.SymbolDisplayVisitor.
```

Comment:
is there any VB impact to `scoped` parameters in general? I think VB doesn't have ref returns, and it doesn't have ref structs, so maybe not?

Reply:
We can consider representing in VB symbol display in the future if there's a scenario.